jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -59,7 +59,7 @@ reasonable, reporting issues to these providers as required.</description>
             categoryId="org.jboss.tools.central.discovery.a.web"
             groupId="org.jboss.tools.central.discovery.a.web.core"
             certificationId="com.jboss.jbds.discovery.certification.supported"
-            description="AngularJS Tooling support for Ionic framework"
+            description="AngularJS Tooling provides content assist and navigation + Ionic framework support"
```

Comment:
Only EA connector provides navigation. It comes from Eclipse AngularJS plugin (available in EA only). JBT/JBDS provides only AngularJS content assist + ionic support.
So this description should be the same as for jbosstools/org.jboss.tools.central.discovery/plugin.xml:
"AngularJS Tooling provides content assist + Ionic framework support"